spec/models/user_spec.rb

```diff
@@ -1,5 +1,25 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+feature 'check user db table' do
+  let(:user) { FactoryBot.create(:user, email: 'testing@gmail.com') }
```

Comment:
Why do you have this line here?

you should have it within the describe block for factory

Reply:
didn't fully understand, but do now.  I have moved it